pkg/store/bucket.go

```diff
@@ -779,7 +796,6 @@ func (s *BucketStore) addBlock(ctx context.Context, meta *metadata.Meta) (err er
 
 	b, err := newBucketBlock(
 		ctx,
-		log.With(s.logger, "block", meta.ULID),
```

Comment:
Why do we remove the logger from here and pass it into every function call?

Reply:
This bucket block logger doesn't have any context information.